x-pack/plugins/actions/server/plugin.ts

```diff
@@ -224,21 +217,24 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
 
     const usageCollection = plugins.usageCollection;
     if (usageCollection) {
+      registerActionsUsageCollector(
+        usageCollection,
+        core
+          .getStartServices()
+          .then(([_, { taskManager }]) => taskManager as TaskManagerStartContract)
```

Comment:
No need for this `as`, the `core.getStartServices()` call can infer from `CoreSetup<ActionsPluginsStart>` automatically.